python/cog/director/director.py

```diff
@@ -312,14 +312,25 @@ def _shutdown_model(self):
 
 def _make_local_http_client() -> requests.Session:
     session = requests.Session()
-    adapter = HTTPAdapter(
-        max_retries=Retry(
-            total=3,
-            backoff_factor=0.1,
-            status_forcelist=[429, 500, 502, 503, 504],
-            allowed_methods=["POST"],
-        ),
-    )
+    auth_token = os.environ.get("WEBHOOK_AUTH_TOKEN")
+    adapter = WebhookHTTPAdapter(auth_token)
```

Comment:
Ah, slight misunderstanding here. This is the HTTP client used to talk to the model container. It looks to me like we use a different client for webhook sending (grep for `webhook_sender`) and for reporting setup runs. I suspect there's a bit of reorganization needed to get both of those using this token.

Reply:
Ah right, thank you!